ballerina/tests/redis_hash_command_test.bal

```diff
@@ -14,49 +14,46 @@
 // specific language governing permissions and limitations
 // under the License.
 
-import ballerina/jballerina.java;
 import ballerina/test;
 
-@test:Config {
-}
+@test:Config {}
 function testHDel() {
-    var result = conn->hDel("testHDelKey", ["testHDelField1", "testHDelField2", "testHDelField3"]);
-    if (result is int) {
+    do {
```

Comment:
Do we need this `do-on-fail` block. I think what this is bit redundant. IMO we can use following structure.
```ballerina
function testHDel() returns error? {
    int result = check redis->hDel("testHDelKey", ["testHDelField1", "testHDelField2", "testHDelField3"]);
    test:assertEquals(result, 3);
    boolean hExistsResult = check redis->hExists("testHDelKey", "testHDelField1");
    boolean hExistsResult2 = check redis->hExists("testHDelKey", "testHDelField2");
    boolean hExistsResult3 = check redis->hExists("testHDelKey", "testHDelField3");
    test:assertTrue(!hExistsResult && !hExistsResult2 && !hExistsResult3); 
}
```

Reply:
+1. Changed with https://github.com/ballerina-platform/module-ballerinax-redis/pull/195/commits/c8aafb0eb9cd47228d3d6ad236a31cea4ee35a5c